pkg/operator/sync_v310_00.go

```diff
@@ -23,6 +23,100 @@ import (
 
 // most of the time the sync method will be good for a large span of minor versions
 func sync_v310_00_to_00(c ServiceCertSignerOperator, operatorConfig *scsv1alpha1.ServiceCertSignerOperatorConfig, previousAvailability *operatorsv1alpha1.VersionAvailablity) (operatorsv1alpha1.VersionAvailablity, []error) {
+	signingVersionAvailability, signingErrors := sync_v310_00_to_00_SigningController(c, operatorConfig, previousAvailability)
```

Comment:
@deads2k the 'to_00' still looks strange to me ;-) I wish we can give it a name like'_to_latest' or something

Reply:
> @deads2k the 'to_00' still looks strange to me ;-) I wish we can give it a name like'_to_latest' or something

will do